easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+versionsuffix = '-Python-%(pyver)s-PyTorch-1.4.0'
```

The versionsuffix can be removed as this easyconfig should have the same dependency on PyTorch as torchvision-0.4.2-fosscuda-2019b-Python-3.7.4.eb, I explain in more detail below
```suggestion
versionsuffix = '-Python-%(pyver)s'
```